package.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "sentry-cli",
+  "version": "0.25.0",
+  "description": "A command line utility to work with Sentry. https://docs.sentry.io/hosted/learn/cli/",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "preinstall": "INSTALL_DIR=./bin ./bin/install-sentry-cli"
```

Comment:
I'm okay with having the duplicated script here but we should strip it down to the bare minimum. In particular the sudo stuff does not belong there any more and we should depend on `npm_package_version`.